check_stability.py

```diff
@@ -353,6 +353,57 @@ def get_files_changed():
             for item in files[:-1].split("\0")]
 
 
+def get_affected_testfiles(files_changed):
+    affected_testfiles = []
+    # FIXME: treat ".xml" files as test files? or support files? or both? or neither?
+    testfile_extensions = [".html", ".htm", ".xhtml", ".xht", ".xml", ".svg"]
```

Comment:
> Couldn't we use the manifest to work out what's a test file and what's a support file?

See https://github.com/w3c/web-platform-tests/pull/4100/commits/987f506fb1b31c26490361034ef1d2f749720968, which is an attempt at that.

As I mentioned on IRC I still wonder why the manifest contains `.worker.js` files and `.any.js` files, and what tests if any the stability checker runs of those change.